### PR TITLE
Including Grafana Links

### PR DESCRIPTION
**Pull request by MVarshini**
## Type of change

- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Based on the systems build Grafana URL and display it on row expansion in OCP page table

## Related Tickets & Documents

- Related Issue #
- [PANDA-424](https://issues.redhat.com/browse/PANDA-424)
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: local-compose.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ CPT_CONFIG=${CPT_CONFIG:-"$PWD/backend/ocpperf.toml"}
 podman rm -f front back
 
 podman build -f backend/backend.containerfile --tag backend
-podman build -f frontend/frontend.containerfile --tag frontend
+# podman build -f frontend/frontend.containerfile --tag frontend
```
Comment (vishnuchalla) [inline local-compose.sh:14]: For testing this PR perhaps?

File: frontend/src/helpers/Formatters.js [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+  else d = new Date(date.props.value);
 
-    var year = d.getFullYear();
-    var month = ("0" + (d.getMonth() + 1)).slice(-2);
-    var day  = ("0" + (d.getDate())).slice(-2);
-    var hour =  ("0" + (d.getHours())).slice(-2);
-    var min =  ("0" + (d.getMinutes())).slice(-2);
-    var sec = ("0" + (d.getSeconds())).slice(-2);
-    return year + "-" + month + "-" + day + " " + hour + ":" +  min + ":" + sec;
+  let year = d.getFullYear();
```
Comment (vishnuchalla) [inline frontend/src/helpers/Formatters.js:22]: just curious, do it matter between `var` vs `let`?

**Review by vishnuchalla — APPROVED** [review]
lgtm

File: frontend/src/helpers/Formatters.js (hunk 2 above)
Comment (MVarshini) [reply frontend/src/helpers/Formatters.js:22]: Using `var` variables can be re-declared and updated within the same scope but with `let` variables can be updated but can't be re-declared in the same scope.

**Pull request merged**